www/webapp/src/views/CrudList.vue

```diff
@@ -170,6 +171,7 @@
                                 :outlined="!createDialogSuccess"
                                 :disabled="createDialogWorking"
                                 @click.native="close"
+                                aria-label={{ createDialogSuccess ? 'Close' : 'Cancel' }}
```

Comment:
Is that valid syntax? I guess it should be:
```suggestion
                                :aria-label="createDialogSuccess ? 'Close' : 'Cancel'"
```